src/SFA.DAS.AssessorService.Data.IntegrationTests/StandardsGetTests.cs

```diff
@@ -43,7 +43,7 @@ public async Task GetStandardByStandardReferenceAndVersion_ReturnsCorrectStandar
         {
             var expectedStandard = _standards.Single(s => s.StandardUId == standardUId);
 
-            var standard = await _repository.GetStandardByStandardReferenceAndVersion(standardReference, version);
+            var standard = await _repository.GetStandardVersionByIFateReferenceNumber(standardReference, version);
 
             Assert.AreEqual(expectedStandard.StandardUId, standard.StandardUId);
         }
```

Comment:
Does this test need a case where it gets the correct standard when no version is supplied?

Reply:
Have added a test case for version is empty to get the latest version